analyzer/codechecker_analyzer/analyzers/clangsa/version.py

```diff
@@ -36,7 +36,7 @@ class ClangVersionInfoParser(object):
 
     def __init__(self):
         self.clang_version_pattern = (
-            r'clang version (?P<major_version>[0-9]+)'
+            r'(?P<vendor>clang|Apple LLVM) version (?P<major_version>[0-9]+)'
```

Comment:
Maybe we can add this (`vendor`) to the `ClangVersionInfo` class:
https://github.com/Ericsson/codechecker/blob/cc93331a92261180cfc9a4ed29cf4d6e9b74ad0d/analyzer/codechecker_analyzer/analyzers/clangsa/version.py#L16-L28